tao_compiler/mlir/disc/transforms/lhlo_legalize_roots_to_loops.cc

```diff
@@ -4098,6 +4098,292 @@ LogicalResult lowerWithScheduleSparseFillEmptyRowsOpCPU(
   return success();
 }
 
+LogicalResult lowerWithScheduleSparseSegmentMeanOpCPU(
```

Comment:
nit: Could all sparse segment reduction op reuse skeleton of this function?

Reply:
It is possible, only the reduction part is different